client/src/componets/ADDVIDEO/AddVideo.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
+import { Container, Form, Row, Col } from "react-bootstrap";
+import Button from "react-bootstrap/Button";
+
+const AddVideo = ({ setVideos }) => {
+  const [addingVideo, setAddingVideo] = useState(false);
+  const [title, setTitle] = useState("");
+  const [url, setUrl] = useState("");
+
+  const VideoAdder = (e) => {
+    e.preventDefault();
+    const videoId = Math.floor(Math.random() * 1000000);
+    const rating = 0;
+
+    const newData = {
+      id: videoId,
+      title: title,
+      url: url,
+      rating: rating,
+      timeSent: new Date().toLocaleDateString(),
+    };
+
+   if(newData.title !=="" && newData.url.includes("www.youtube.com/watch?v=")){
+   setVideos((videos) => videos.concat(newData))
```

Comment:
👀 Be mindful of the indenting here. How can you make sure your code is formatted properly?

Reply:
Thank you , i have used Preittier extention to format it.